week3/labs/lab3-1/public/js/a4.js

```diff
@@ -0,0 +1,35 @@
+var numbers = [1,2,3,4,5,6,7,8,9,10,11]
+
+for (i = 0; i < 11; i++){
+  console.log("i = " + i);
+}
+
+numbers.forEach(function(e){
+  console.log("number " + e);
+})
+
+for (i = 0; i < numbers.length; i++){
+  if (numbers[i] > 5)
+    console.log("larger than 5: " + numbers[i]);
```

Comment:
please always use `{ ... }` when using `if` to prevent accidental mistakes:

```
if (numbers[i] > 5) {
}
```

note please also put `var i = 0` instead of `i = 0`

by the way, just for fun, the code is faster if you only calculate `numbers.length` once

```
for(var i = 0, len = numbers.length; i < len; i++)
```